src/Wallabag/CoreBundle/Filter/EntryFilterType.php

```diff
@@ -39,10 +39,23 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         $expression = $filterQuery->getExpr()->like($field, $filterQuery->getExpr()->literal('%'.$value.'%'));
 
                         return $filterQuery->createCondition($expression);
-                    },
+                },
             ))
             ->add('isArchived', 'filter_checkbox')
-            ->add('isStarred', 'filter_checkbox');
+            ->add('isStarred', 'filter_checkbox')
+            ->add('previewPicture', 'filter_checkbox', array(
+                'apply_filter' => function (QueryInterface $filterQuery, $field, $values) {
+                    $value = $values['value'];
```

Comment:
Since you won't use `$value` after, you can directly use `$values['value']` in your `if` case.